app/models/setting/rh_cloud.rb

```diff
@@ -7,7 +9,7 @@ def self.default_settings
       set('allow_auto_insights_sync', N_('Allow recommendations synchronization from Red Hat cloud'), false),
       set('obfuscate_inventory_hostnames', N_('Obfuscate host names sent to Red Hat cloud'), false),
       set('obfuscate_inventory_ips', N_('Obfuscate ip addresses sent to Red Hat cloud'), false),
-      set('rh_cloud_token', N_('Authentication token to Red Hat cloud services. Used to authenticate requests to cloud APIs'), 'DEFAULT', N_('Red Hat Cloud token'), nil, encrypted: true),
+      set('rh_cloud_token', N_('Authentication token to Red Hat cloud services. Used to authenticate requests to cloud APIs'), nil, N_('Red Hat Cloud token'), nil, encrypted: true),
```

Comment:
Do we need a migration for that? @ShimShtein ?

Reply:
When I added `::Setting::BLANK_ATTRS.concat %w{rh_cloud_token}` I didn't need a migration for it to get the new default